pkg/cloudprovider/vsphereparavirtual/cloud.go

```diff
@@ -125,7 +125,6 @@ func (cp *VSphereParavirtual) Initialize(clientBuilder cloudprovider.ControllerC
 	cp.routes = routes
 
 	cp.informMgr.AddNodeListener(cp.nodeAdded, cp.nodeDeleted, nil)
-	cp.informMgr.Listen()
```

Comment:
why should we remove this?
if we intend to remove this, should we not call `cp.informMgr.AddNodeListener`?

Reply:
https://github.com/kubernetes/cloud-provider-vsphere/blob/master/pkg/cloudprovider/vsphereparavirtual/cloud.go#L150
Because there is another one here, I intended to call it after the handler registration inside `StartControllers`